mono/utils/w32handle.h

```diff
@@ -165,10 +167,11 @@ int
 mono_w32handle_unlock_handle (gpointer handle);
 
 typedef enum {
-	MONO_W32HANDLE_WAIT_RET_SUCCESS_0 =  0,
-	MONO_W32HANDLE_WAIT_RET_ALERTED   = -1,
-	MONO_W32HANDLE_WAIT_RET_TIMEOUT   = -2,
-	MONO_W32HANDLE_WAIT_RET_FAILED    = -3,
+	MONO_W32HANDLE_WAIT_RET_SUCCESS_0   =  0,
+	MONO_W32HANDLE_WAIT_RET_ABANDONED_0 =  0x80,
```

Comment:
It should be `MONO_W32HANDLE_WAIT_RET_SUCCESS_0 + MONO_W32HANDLE_MAXIMUM_WAIT_OBJECTS`